.ruby-version

```diff
@@ -1 +1 @@
-2.6.3
\ No newline at end of file
+3.0.2
```

Comment:
Updates Ruby to 3.0.2 since `liquid-c` was failing on my machine with Ruby 2.x

Reply:
Should we bump up `.bundler-version` too? Last update was 2 years ago and latest is `2.2.0`.

https://github.com/uswds/uswds-site/blob/b961cbd6f99ff63e16ed450631c051f5b152fc84/.bundler-version#L1